Sources/Formworks/Design System/FWStyle.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 /// The form's color style.
-public enum FWStyle {
+public enum FWStyle: Equatable {
 	/// The default light color style of a form.
 	case light
 	
```

Comment:
Review FWStyle and make necessary changes to make it work with JSON. With the current version, the JSON will always override it.